servatrice/src/serversocketinterface.cpp

```diff
@@ -164,14 +164,24 @@ void ServerSocketInterface::readClient()
             newCommandContainer.ParseFromArray(inputBuffer.data(), messageLength);
         }
         catch(std::exception &e) {
-            qDebug() << "Caught std::exception in" << __FILE__ << __LINE__ << __PRETTY_FUNCTION__;
+            qDebug() << "Caught std::exception in" << __FILE__ << __LINE__ << 
+#ifdef _MSC_VER // Visual Studio
+                __FUNCTION__;
+#else
+                __PRETTY_FUNCTION__;
+#endif
```

Comment:
Do you think this might get used anywhere else? I wonder if it's not better to `#DEFINE` just one of these and use it throughout.